src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import CountdownTimer from "../../components/CountdownTimer";
+
+const Home = () => (
+  <div className="Home" data-test="Home-page">
+    <h1>SnapIT Hacktoberfest Starts In:</h1>
```

Comment:
:wrench: add a test for this heading

Reply:
Addressed:  52d3b149150f2810f78e3a2c2c6780dd74185b0c